cmd/main.go

```diff
@@ -74,7 +73,43 @@ func main() {
 		&config.PluginType, "plugin", "csi-plugin", "Type of this driver i.e. controller or node",
 	)
 
+	cmd.PersistentFlags().BoolVar(
+		&config.SetIOLimits, "setiolimits", false,
+		"Whether to set iops, bps rate limit for pods accessing volumes",
+	)
+
+	cmd.PersistentFlags().StringVar(
+		&config.ContainerRuntime, "container-runtime", "containerd",
+		"Whether to set iops, bps rate limit for pods accessing volumes",
+	)
+
+	config.RIopsLimitPerGB = cmd.PersistentFlags().StringSlice(
+		"riops-per-gb", []string{},
+		"Read IOPS per GB limit to use for each volume group prefix, "+
+			"--riops-per-gb=\"vg1-prefix=100,vg2-prefix=200\"",
+	)
+
+	config.WIopsLimitPerGB = cmd.PersistentFlags().StringSlice(
```

Comment:
this will be based on disktype. One node agent could serve more than one class of disktype. Getting iops  per Gb is not the right thing, we should be getting maximum iops per storage class/ storage pool type

Reply:
that can also be done. But typically I have seen the iops of a storage class being conveyed in terms of iops/GB (atleast from internal documents at Flipkart IaaS). Hence this decision. But its not a hard bound decision. If we feel strongly on your suggestion, I can change it. Lets have a discussion on this in the next openebs sync up meet